### PR TITLE
edu-2600-astro-boilerplate-guide

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Changes

Adding new Astro boilerplate guide.

### Additional links
-

[EDU-2600]: https://aziontech.atlassian.net/browse/EDU-2600?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/guides/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
+title: How to deploy edge applications with the Astro Boilerplate
+description: The Astro Boilerplate provides an automation solution to build a Astro application on Azion.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx:3]:
```suggestion
description: The Astro Boilerplate provides an automation solution to build an Astro application on Azion.
```

File: src/content/docs/en/pages/guides/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+permalink: /documentation/products/guides/astro-boilerplate/
+---
+
+import Badge from '~/components/Badge.astro';
+
+<Badge variant="accent">
+Preview
+</Badge>
+
+The Azion **Astro Boilerplate** helps you build a **Astro Single-Page Application (SPA)** and run it directly on the edge of the network. It encapsulates and automates several steps, from repository management to edge deployment.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx:15]:
```suggestion
The Azion **Astro Boilerplate** helps you build an **Astro Single-Page Application (SPA)** and run it directly on the edge of the network. It encapsulates and automates several steps, from repository management to edge deployment.
```

File: src/content/docs/en/pages/guides/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+:::note
+The link to the edge application allows you to see it on the browser. However, ​​it takes a certain time to propagate and configure the application in Azion’s edge locations. It may be necessary to wait a few minutes for the URL to be activated and for the application page to be effectively displayed in the browser.
+:::
+
+### Key configurations
+
+The application undergoes an optimized build process, emphasizing optimal performance and efficiency, using the resources of **Azion Edge Platform**.
+
+The script creates a [GitHub repository](https://github.com/aziontech/azion-samples/tree/dev/templates/astro-boilerplate) to manage the source code and then it installs all required dependencies. Besides that, your application is deployed on the Azion's platform and includes multiple configurations, such as setting up the edge application and its domain to host your Astro project and an edge function for custom logic.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx:72]:
```suggestion
The script creates a [GitHub repository](https://github.com/aziontech/azion-samples/tree/dev/templates/astro-boilerplate) to manage the source code and then it installs all required dependencies. Besides that, your application is deployed on Azion's platform and includes multiple configurations, such as setting up the edge application and its domain to host your Astro project and an edge function for custom logic.
```

File: src/content/docs/en/pages/guides/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+Considering that this initial setup may not be optimal for your specific edge application, all settings can be customized any time you need by using Azion Real-Time Manager (RTM).
+
+To manage and edit your edge application's settings, proceed as follows:
+
+1. [Access RTM](https://manager.azion.com/).
+2. On the upper-left corner, select the three horizontal lines to open the **Products menu**.
+3. In the **BUILD** section, select **Edge Application**.
+4. You'll be redirected to the **Edge Application** page.
+- It lists all the edge applications you've created. 
+5. Find the edge application relate to the Astro Boilerplate and select it.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx:111]:
```suggestion
5. Find the edge application related to the Astro Boilerplate and select it. 
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+Este boilerplate usa o Astro versão `2.6.4`.
+
+---
+
+## Obter o template
+
+Para começar a usar o **Astro Boilerplate**, siga estes passos:
+
+1. Acesse o [Real-Time Manager (RTM)](https://manager.azion.com/).
+- Se você ainda não tem uma conta, crie uma [visitando a página de signup](https://manager.azion.com/signup/).
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx:26]:
```suggestion
- Se você ainda não tem uma conta, crie uma [visitando a página de sign up](https://manager.azion.com/signup/).
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+- A aba **Deployment details** pode ser aberta para acessar os logs relacionados à implantação.
+- **What do you want to do next?** fornece recomendações sobre opções avançadas para aprimorar sua edge application: **Customize Domain**, **Manage Edge Application**, **View Edge Application Metrics**, **View Edge Application Logs** e o botão **Back to Home**.
+
+:::note
+O link para sua edge application permite que você veja como ela fica no navegador. No entanto, leva um certo tempo para propagar e configurá-la nas edge locations da Azion. Pode ser necessário aguardar alguns minutos para que a URL seja ativada e para que a página da aplicação seja efetivamente exibida no navegador.
+:::
+
+### Principais configurações
+
+A aplicação passa por um processo de criação otimizado, enfatizando no melhor desempenho e a eficiência, aproveitando os recursos da **Plataforma de Edge da Azion**.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx:70]:
```suggestion
A aplicação passa por um processo de criação otimizado, enfatizando um melhor desempenho e eficiência, aproveitando os recursos da **Plataforma de Edge da Azion**.
```

**Review by hannah-sk — APPROVED** [review]
deixei pequenas correções

File: src/content/docs/pt-br/pages/guias/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+1. [Acesse o RTM](https://manager.azion.com/).
+2. No canto superior esquerdo, selecione as três linhas horizontais para abrir o **Products menu**.
+3. Na seção de **BUILD**, selecione **Edge Application**.
+4. Você será redirecionado para a página de **Edge Application**.
+- Ela lista todas as edge applications que você criou.
+5. Encontre a aplicação relacionada ao seu template e selecione-a.
+6. Na aba **Main Settings**, encontre a seção **Edge Application Modules** e ative o switch do produto.
+
+:::caution[Atenção]
+Se esse produto for ativado, a execução deste template poderá gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para obter mais informações.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/astro-boilerplate/astro-boilerplate.mdx:95]:
```suggestion
Se esse produto for ativado, a execução deste template poderá gerar custos relacionados ao uso. Consulte a [página de Preços](https://www.azion.com/pt-br/precos/) para obter mais informações.
```

**Review by bru-andrade — APPROVED** [review]
👯‍♀️

**Pull request merged**